### ISSUE TITLE
Dokončana projektna naloga

### ISSUE THREAD

**Issue by ian-spiller** [ian-spiller/APPR-2020-21#1]
Pozdravljeni,

končal sem s projektno nalogo. Prosim vas, če mi lahko sporočite ali je moja naloga v redu narejena.

Že v naprej se vam lepo zahvaljujem in vas lepo pozdravljam,
Ian Spiller

**Comment by jaanos** [comment]
Poročilo deluje, tako da bom dodal povezavo na repozitorij za zagovore. Če se še nisi, se lahko z @alenFMF domeniš za termin zagovora. Če boš do takrat še kaj spreminjal, mi javi, da posodobim povezavo.

Imam še sledeče pripombe:
* Na tortnem diagramu marže za S&P500 je besedilo v manjšem delu težko berljivo. Svetujem, da spremeniš barvo na temnejšo (ali pa nasploh zamenjaš barve tako, da bo kontrast večji).
* Pri napovedih z linearnim modelom pri `geom_smooth` dodaj parameter `fullrange=TRUE`, da se bo trendna črta razširila v območje napovedi (zaželeno bi bilo tudi pri uporabi `loess`, a to žal ne deluje).
* Pri vključevanju aplikacije Shiny v poročilo povečaj vrednost parametra `height`, da se znebiš navpičnega drsnika. Poleg tega svetujem, da v `ui.R` namesto `navbarPage` uporabiš `tabPanel`, ki naj bo znotraj `tabsetPanel`:
  ```R
  shinyUI(
    fluidPage(
      theme = shinytheme("darkly"),
      tabsetPanel(
        tabPanel("AAPL",
                 ...
        )
      )
    )
  )
  ```
  Tako se tudi znebiš neželenega besedila nad stranskim stolpcem

Mimogrede, poročilo se trenutno ne prevede na Binderju, saj v osnovni sliki še ni knjižnice `Quandl`. Sem zdaj dal delat novo sliko z omenjeno knjižnico - ko bo pripravljena (morala bi biti v roku ene ure), se bo zraven zgoraj omenjenega commita pojavila zelena kljukica. Če boš naredil commit za tem (oziroma prvič pognal prevajanje na Binderju), bi moralo zagrabiti novo sliko, na kateri bo poročilo delovalo.

**Comment by ian-spiller** [comment]
Najlepša hvala. Sem popravil zgoraj navedene napake. Upam, da je sedaj bolje.

**Comment by jaanos** [comment]
V redu, vidim, da si sprejel pull request #2, tako da bom dodal povezavo do trenutne različice projekta.

**Comment by ian-spiller** [comment]
Najlepša hvala.

**Comment by ian-spiller** [comment]
Oprostite,  sedaj, ko se pripravljam na zagovor nisem več prepričan, da je tabela `morningstar` v mojem programu v obliki tidy data. Saj imam stolpce `Leto`, `Dobicek` , `Prodaja` itd. , kjer se vsako leto izmerijo vse te meritve. Ali bo morala biti tabela oblike `Leto`, `Ime meritve` , `Vrednost`.?

Že v naprej hvala za odgovor.

**Comment by katarinabrilej** [comment]
Ja, v tidy data bi bila tabela oblike: leto, ime meritve, vrednost.

**Comment by ian-spiller** [comment]
Najlepša hvala. Ali je najbolje, da popravim program in ga še enkrat naložim na github?

**Comment by jaanos** [comment]
Lahko popraviš, pa javi, ko bo urejeno, da posodobim povezavo.

**Comment by ian-spiller** [comment]
Najlepša hvala sem popravil.

**Comment by jaanos** [comment]
V redu, sem posodobil povezavo.
